### DISCUSSION TITLE
Styling of ESRI-Feature Layer

### DISCUSSION THREAD

**Discussion by leylines** [TerriaJS/terriajs#7107] [Q&A]
Hi,

I have a Feature Layer from ESRI that i want to show in my catalog. There is a "Drawing Info" for this Layer with colors, but all the polygons appear in the same color in TerriaMap. Is there a way to show the colors from the Drawing Info for this layer?

https://services1.arcgis.com/qN3V93cYGMKQCOxL/arcgis/rest/services/DARMC_Medieval_World/FeatureServer/5

Kind regards,

Jörg

**Comment by mwu2018** [comment]
You can use the "Edit Style" menu (click on the 3-dot icon next to the "ABOUT DATA" button) to set your custom styles. E.g. https://nationalmap.gov.au/#share=s-kE6feCTYDFj8PWFRaj11vBAB4Iq
